Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/fubar/ubuntu-indexer
```

Comment:
I think I'd like to flip this upside-down. Since the dev env is a subset of what's needed on the indexers, it seems to me that the indexer should pull from this, rather than the other way around. We should move things like [virtualenv setup](https://github.com/klibby/dxr-docker/blob/master/docker/ubuntu-indexer/bin/dxr-setup.sh#L35-L36) over to this, and the Jenkins-specific stuff can stay over on the indexer image. How does that strike you?